components/Drawer.tsx

```diff
@@ -0,0 +1,99 @@
+import Drawer from '@mui/material/Drawer'
+import List from '@mui/material/List'
+import ListItemButton from '@mui/material/ListItemButton'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import ListItemText from '@mui/material/ListItemText'
+import { Box } from '@mui/system'
+import StarIcon from '@mui/icons-material/Star'
+import Image from 'next/image'
+import Collapse from '@mui/material/Collapse'
+import ExpandLess from '@mui/icons-material/ExpandLess'
+import ExpandMore from '@mui/icons-material/ExpandMore'
+import React from 'react'
+import AccessTimeIcon from '@mui/icons-material/AccessTime'
+import SettingsIcon from '@mui/icons-material/Settings'
+
+export default function NavigationDrawer() {
+  // Params for the Drawer
+  const drawerWidth = 280
+  // Logo definition //TODO placeholder
+  const logo = (
+    <Image
+      src="/../public/images/CCAHT-Logo.png"
+      alt="CCAHT-logo"
+      height="70"
+      width="239"
+    />
+  )
+  // state handlers for expandable lists in drawer
+  const [open, setOpen] = React.useState(true)
+  const handleClick = () => {
+    setOpen(!open)
+  }
+  // Drawer setup
+  const drawer = (
+    <Box>
+      <Box sx={{ pr: 3, mt: 5 }}>{logo}</Box>
+      <br />
+      <ListItemButton>
```

Comment:
A good way to simplify this would be to define another component for one of these button rows, that way we cut down on duplicated code. It'd probably take in an icon, display text, and children (for the case of settings) as props. Let me or Zavier know if you have any questions about this or if you want to pair. We can also get you started in our next standup.